pkg/git/repospec_test.go

```diff
@@ -163,6 +163,12 @@ func TestNewRepoSpecFromUrl_CloneSpecs(t *testing.T) {
 			path:  "",
 			ref:   "v0.1.0",
 		},
+		{
+			input: "git@bitbucket.org:company/project.git//path?ref=branch",
+			repo:  "git@bitbucket.org:company/project.git",
+			path:  "//path",
```

Comment:
Should it be `path` instead of `//path`?

Reply:
//path is the right value here, but it's not easy to see that.

Changed the test to check AbsPath(), a public method and more relevant to what actually happens.